packages/block-editor/src/components/inspector-controls/block-support-tools-panel.js

```diff
@@ -11,46 +11,80 @@ import { store as blockEditorStore } from '../../store';
 import { cleanEmptyObject } from '../../hooks/utils';
 
 export default function BlockSupportToolsPanel( { children, group, label } ) {
-	const { clientId, attributes } = useSelect( ( select ) => {
-		const { getBlockAttributes, getSelectedBlockClientId } = select(
-			blockEditorStore
-		);
+	const { attributes, clientIds, panelId } = useSelect( ( select ) => {
+		const {
+			getBlockAttributes,
+			getMultiSelectedBlockClientIds,
+			getSelectedBlockClientId,
+			hasMultiSelection,
+		} = select( blockEditorStore );
+
+		// This is `null` if multi-selection and used in `clientId` checks
+		// to still allow panel items to register themselves.
```

Comment:
Could you explain this comment to me? 

Does it mean `selectedBlockClientId` is `null` when `hasMultiSelection() === true` and ... 

or should that "and" be an "is"?

Reply:
Update: I probably should have started looking at https://github.com/WordPress/gutenberg/pull/37273 before this one 😅

Reply:
Apologies for the confusion. 

I've pushed a commit with an updated comment hopefully clarifying how this works.

The PR was split to isolate the `ToolsPanel` related changes as has been requested in the past.